### PR TITLE
Adding snippets for Payment Api

### PR DESCRIPTION
**Pull request by KBbitsP**
Added Python snippets for Payment Api

### CODE REVIEW

File: source/includes/_payment.md [hunk 1]
```diff
@@ -936,14 +1023,20 @@ curl -v \
...
 PaymentTransaction body = new PaymentTransaction();
 body.setPaymentExternalKey("a975f7b8-1e59-4801-91c9-fcce1526b019");
 body.setAmount(new BigDecimal(5));
 		
 List<String> NULL_PLUGIN_NAMES = null;
 Map<String, String> NULL_PLUGIN_PROPERTIES = null;
 		
-Payment payment = paymentApi.chargebackPaymentByExternalKey(body, NULL_PLUGIN_NAMES, NULL_PLUGIN_PROPERTIES, requestOptions);
+Payment payment = paymentApi.chargebackReversalPayment(payment_id, 
+                                                       body,
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1036]: Is this code alignment correct?

File: source/includes/_payment.md [hunk 2]
```diff
@@ -1836,16 +2032,22 @@ payment.add_tag(tag_name,
 ```
 
 ```python
-paymentApi = killbill.api.PaymentApi()
+created_by = 'user'
+
+paymentApi = killbill.PaymentApi()
 tag = ["353752dd-9041-4450-b782-a8bb03a923c8"]
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2038]: Consider renaming to `tagDefIds`

File: source/includes/_payment.md [hunk 3]
```diff
@@ -1979,15 +2188,23 @@ payment.remove_tag(tag_name,
...
 
 ```python
-paymentApi = killbill.api.PaymentApi()
+created_by = 'user'
+
+paymentApi = killbill.PaymentApi()
+
 payment_id = 'dce5b2a0-0f0f-430b-9427-545ba4be5c7f'
-tag = ["353752dd-9041-4450-b782-a8bb03a923c8"] 
+tagDefId = ["353752dd-9041-4450-b782-a8bb03a923c8"]
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2196]: If this is an array, consider renaming to `tagDefIds`

File: source/includes/_payment.md [hunk 4]
```diff
@@ -261,12 +280,19 @@ KillBillClient::Model::Payment.find_by_id(payment_id,
 ```
 
 ```python
-paymentApi = killbill.api.PaymentApi()
+paymentApi = killbill.PaymentApi()
+
 payment_id = 'ce88ae5b-7ec0-4e14-9ea1-fffe4411278e'
 
-paymentApi.get_payment(payment_id, api_key, api_secret)
+paymentDetails = paymentApi.get_payment(payment_id)
```
Comment (reshmabidikar) [inline source/includes/_payment.md:287]: Consider renaming to `payment`

File: source/includes/_payment.md [hunk 5]
```diff
@@ -373,14 +400,19 @@ KillBillClient::Model::Payment.find_by_external_key(external_key,
...
 ```python
-paymentApi = killbill.api.PaymentApi()
+paymentApi = killbill.PaymentApi()
+
 payment_external_key = 'sample_external_key' 
 
-paymentApi.get_payment_by_external_key(payment_external_key,
-                                       api_key,
-                                       api_secret)
+paymentDetails = paymentApi.get_payment_by_external_key(payment_external_key)
```
Comment (reshmabidikar) [inline source/includes/_payment.md:407]: Consider renaming to `payment`

File: source/includes/_payment.md [hunk 6]
```diff
@@ -1761,13 +1949,21 @@ payment.remove_custom_field(custom_field_id,
...
+paymentApi = killbill.PaymentApi()
+
 payment_id = 'f33e0adc-78df-438a-b920-aaacd7f8597a'
 
 paymentApi.delete_payment_custom_fields(payment_id,
-                                        created_by,
-                                        api_key,
-                                        api_secret)
+                                        created_by)
+```
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1960]: I think the `customField` of the field to be deleted also needs to be specified here. Can you confirm and if so add this parameter? See [this thread.](https://killbillio.slack.com/archives/C02C15Z92PJ/p1692775418598969)

**Review by reshmabidikar — COMMENTED** [review]
Looks good, left some minor comments.

File: source/includes/_payment.md (hunk 1 above)
Comment (KBbitsP) [reply source/includes/_payment.md:1036]: Hi Reshma, yup its correct.

image

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**